tests/migrations/test_state.py

```diff
@@ -870,6 +871,38 @@ class Meta:
         self.assertEqual([name for name, mgr in food_state.managers], ['food_mgr'])
         self.assertEqual(food_state.managers[0][1].args, ('a', 'b', 1, 2))
 
+    def test_metaclass_conflict(self):
+        """
+        Test making class with complex metadata inheritance
+        """
+        new_apps = Apps(['migrations'])
+
+        class DummyMetaClass(type):
+            dummy_attr = 'dummy'
+            pass
```

Comment:
hehe, you are right :) You want to tire me with this topic to death ;) will fix in minutes.